src/utils/RestfulUtils.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Restful Utils
+ * Define common utils used by controllers
+ */
+
+import * as express from "express";
+import { ControllerError } from "../enums/ControllerError";
+import { HttpErrorStatusCode } from "../enums/HttpErrorStatusCode";
+import { IRestfulObject } from "../types/CommonTypes";
+import { logger } from "../utils/Logger";
+
+// Send an error message for express response
+export function sendErrorResponse(
+  res: express.Response,
+  errorMsg: ControllerError,
+  httpStatusCode: HttpErrorStatusCode
+): ControllerError {
+  logger.error(`Controller response is an error: ${errorMsg}`);
+  res.status(httpStatusCode).json({ errorMessage: errorMsg });
+  return errorMsg;
+}
+
+// Send a success message for express response
+export function sendSuccessResponse(
```

Comment:
use [this](https://github.com/teamdigitale/italia-ts-commons/blob/master/src/responses.ts#L56) instead